prov/sockets/src/sock_conn.c

```diff
@@ -467,8 +466,10 @@ struct sock_conn *sock_ep_connect(struct sock_ep_attr *ep_attr, fi_addr_t index)
 	if (!do_retry)
 		goto err;
 
-	close(conn_fd);
-	conn_fd = -1;
+	if (conn_fd != -1) {
+		ofi_close_socket(conn_fd);
+		conn_fd = -1;
+	}
```

Comment:
during porting to windows i catched situation when conn_fd was not properly processed & has value -1